### PR TITLE
fix: use type alias instead of type definition

### PR DESCRIPTION
**Pull request by CaptainStandby**
## Related issue(s)

https://github.com/ory/network/issues/251

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

File: schema/handler.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func (h *Handler) RegisterAdminRoutes(admin *x.RouterAdmin) {
 //
 //nolint:deadcode,unused
 //lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
-type identitySchema json.RawMessage
+type identitySchema = json.RawMessage
```
Comment (jonas-jonas) [inline schema/handler.go:73]: What the...

**Review by jonas-jonas — APPROVED** [review]
LGTM.

Would this be a breaking change in the SDK? I guess since the Go SDK didn't change (apart from the doc comment) no? What does this look like in TS?

File: schema/handler.go (hunk 1 above)
Comment (zepatrik) [reply schema/handler.go:73]: When you define a new type (without `=`), it removes all of the underlying types methods. Using `=` means that we only have another name for the same type.

**Review by zepatrik — APPROVED** [review]

File: spec/api.json [hunk 2]
```diff
@@ -971,7 +971,8 @@
             "type": "string"
           },
           "schema": {
-            "$ref": "#/components/schemas/identitySchema"
```
Comment (alnr) [inline spec/api.json:974]: Why this change?

File: schema/handler.go (hunk 1 above)
Comment (zepatrik) [reply schema/handler.go:73]: See also https://go.dev/play/p/rZhjaze9ZYv

Comment (alnr) [reply schema/handler.go:73]: https://go.googlesource.com/proposal/+/master/design/18130-type-alias.md

Comment (CaptainStandby) [reply schema/handler.go:73]: The problem was that `func (m RawMessage) MarshalJSON() ([]byte, error)` and `func (m *RawMessage) UnmarshalJSON(data []byte) error` is dropped when using type definitions which in turn leads to the json marshaller treating `identitySchema` like a byte array and serializing it as a base64 string

File: spec/api.json (hunk 2 above)
Comment (CaptainStandby) [reply spec/api.json:974]: This was generated by `make sdk`. This is a result of using type alias instead of type definition

**Review by alnr — APPROVED** [review]

**Comment by CaptainStandby** [comment]
> LGTM.
> 
> Would this be a breaking change in the SDK? I guess since the Go SDK didn't change (apart from the doc comment) no? What does this look like in TS?

I hope not 🙃
